plutus-benchmark/flat/Dataset.hs

```diff
@@ -35,7 +36,7 @@ wallet2 = Wallet 2
 escrowParams :: Escrow.EscrowParams d
 escrowParams =
   Escrow.EscrowParams
-    { Escrow.escrowDeadline = TimeSlot.slotToPOSIXTime 200
+    { Escrow.escrowDeadline = TimeSlot.slotToEndPOSIXTime def 200
```

Comment:
At some point we should just define these in terms of times. Perhaps what we'll end up wanting is to do something like
```
escrowParams :: Time -> EscrowParams d
escrowParams startTime = ...
```
and then e.g. we could pass in the beginning of the first slot as the start time when we set it up. Otherwise we keep effectively abusing slots as a way of specifying relative times!

Reply:
(not for this PR)

Reply:
Took note of this for a future PR.